source/how-tos/debug/debug-apache.rst

```diff
@@ -77,7 +79,39 @@ Or you're using the wrong hostname in your browser.
 
           sudo /sbin/apache2 -S
 
+Performance Tuning
+------------------
+
+If you're servicing many clients at time (more than 50) you will likely need to change the
+`Apache Httpd's worker configuration`_. The default configuration may degrade service when
+Httpd has to serve many clients (I.e., when you have a lot of customers using Open OnDemand).
+
+We suggest configurations similar to this. 
+
+.. warning:: 
+  `ServerLimit` and `ThreadsPerChild` should not exceed the number of cores available. 8
+  is given in the example, but will likely need to change.
+
+.. code-block:: apache
+
+  # /opt/rh/httpd24/root/etc/httpd/conf.d/mpm.conf
+
+  LoadModule mpm_event_module modules/mod_mpm_event.so
+
+  <IfModule mpm_event_module>
+    ServerLimit            8
```

Comment:
`ServerLimit` and `ThreadsPerChild` should not exceed the number of cores available so I left this to 8 given that may just fit a lot of sites (we have 32).

Reply:
Per comment in Puppet:

```
# This value is maxrequestworkers / threadsperchild then doubled
```

Reply:
Thanks Ive updated all of these.